### PR TITLE
Initial full copyedit, normalization

### PR DESCRIPTION
**Pull request by quaid**
Great chapter, seems very complete. I made a number of edits, including:
* grammar, style, and formatting
* normalizing/standardizing usage (emdash vs parentheses, etc.)
* established a consistent usage of _new term_ for where the _emphasis_ adoc tag is used, rather than "quote marks"
* * in American English, the "scare quotes" effect makes it impossible to use quotes for emphasis
* many instances of emphasis removed where it was to emphasize a word; rewrote where necessary to create the emphasis
* removed idioms, rewrote with plainer language
* reduced some sentence complexity to facilitate comphrension
* other things I blame _Dreyer's English_ for

### CODE REVIEW

File: community_governance.adoc [hunk 1]
```diff
@@ -261,10 +257,11 @@ Do not expect to influence decisions in a do-ocracy until you are able to demons
 The founder-leader governance model is most common among new projects or those with a small number of contributors (and since most open source projects have only a small number of contributors, this is a rather popular model!).
 In these projects, the individual or group who started the project also administers the project, establishes its vision, controls all permissions to merge code into it, and assumes the right to speak for it in public.
 Some projects refer to their founder-leaders as "BDFLs" or "Benevolent Dictators for Life."
+// @quaid I thought we weren't going to make the BDFL reference? Maybe reference it as an archaic form falling out of favor in recent years, etc.?
 
 In projects following the founder-leader model, lines of power and authority are typically quite clear; they radiate from founder-leaders, who are the final decision-makers for all project matters.
 This model's limitations become apparent as a project grows to a certain size.
-Separating the founder-leaders' personal preferences from project design decisions eventually becomes difficult, and founder-leaders can become bottlenecks for project decision-making work.
```
Comment (semioticrobotic) [inline community_governance.adoc:267]: We've decided to retain the plural possessive to best indicate that project founders can indeed be multiple (as in the rest of this section).

File: community_governance.adoc [hunk 2]
```diff
@@ -261,10 +257,11 @@ Do not expect to influence decisions in a do-ocracy until you are able to demons
 The founder-leader governance model is most common among new projects or those with a small number of contributors (and since most open source projects have only a small number of contributors, this is a rather popular model!).
 In these projects, the individual or group who started the project also administers the project, establishes its vision, controls all permissions to merge code into it, and assumes the right to speak for it in public.
 Some projects refer to their founder-leaders as "BDFLs" or "Benevolent Dictators for Life."
+// @quaid I thought we weren't going to make the BDFL reference? Maybe reference it as an archaic form falling out of favor in recent years, etc.?
```
Comment (semioticrobotic) [inline community_governance.adoc:260]: We decided not to omit entirely, but to relegate to the description of the governance model, not make it the label for this model/section. We might even consider specifying further in the language here (something like "... for Life," a term that is falling out of fashion.")

File: community_governance.adoc [hunk 3]
```diff
@@ -381,77 +377,78 @@ If there are aspirational goals, those go in their own section under "Roadmap" o
...
 
 Far more projects have a written definition of Contributors, but fewer than you'd think.
-It's often assumed, in the age of publicly hosted source code control, that you simply count up anyone who shows up in the Github or Gitlab statistics is therefore a Contributor.
+It's often assumed, in the age of publicly hosted source code control, that you count anyone in the GitHub or GitLab statistics as therefore a Contributor.
 But defining "who is a Contributor to this project" can be deceptively hard.
 
 Is it anyone who posted on a mailing list, or do you need 100 merged pull requests?
 Is it just code contributors, or contributors of any kind? What about folks who do events and advocacy?
-Are staff who work for a contributing company automatically considered contributors, or do they have to earn it individually?
+Are staff who work for a contributing company automatically consideredCcontributors, or do they have to earn it individually?
```
Comment (semioticrobotic) [inline community_governance.adoc:420]: Not sure what happened here?

**Review by semioticrobotic — CHANGES_REQUESTED** [review]
Thanks for this thorough read. In general, I have no objectives to the revisions, which better align the chapter with the book's overall style guidelines. Please see three outstanding comments.

File: community_governance.adoc (hunk 3 above)
Comment (quaid) [reply community_governance.adoc:420]: Inadvertent edit. Could be the drifting 'c' that my Bluetooth keyboard sometimes leaves.

I'm unsure what action to take here to fix this, or accept your requested changes. Time to experiment, starting with this comment to see what happens.

File: community_governance.adoc (hunk 2 above)
Comment (quaid) [reply community_governance.adoc:260]: I like the falling out of fashion option.

**Review by quaid — COMMENTED** [review]
@semioticrobotic I fixed things as per your requests, this should be ready to merge.

File: community_governance.adoc (hunk 1 above)
Comment (quaid) [reply community_governance.adoc]: fixed and committed

File: community_governance.adoc (hunk 3 above)
Comment (quaid) [reply community_governance.adoc]: fixed

**Pull request merged**